user-guide/Reference/Soft-launch_options/Overview_of_Soft_Launch_Options.md

```diff
@@ -396,7 +396,9 @@ Configures the DataMiner System to use SNMP++ for the polling of all three SNMP
 - **Release version**: To be determined.
 
 > [!IMPORTANT]
-> This feature does not support polling of IPv6 addresses. In case IPv6 addresses are polled on your DMA, do not activate this feature.
+>
+> - This feature does not support polling of IPv6 addresses. In case IPv6 > addresses are polled on your DMA, do not activate this feature.
```

Comment:
```suggestion
> - This feature does not support polling of IPv6 addresses. In case IPv6 (or higher) addresses are polled on your DMA, do not activate this feature.
```

Reply:
```suggestion
> - This feature does not support polling of IPv6 addresses. In case IPv6 addresses are polled on your DMA, do not activate this feature.
```

My bad @MariekeGO, this '>' was not added on purpose and probably got lost when trying to format the list within the Important section.